public/resources/remix/index.js

```diff
@@ -6,122 +6,126 @@
 
   var gaTrackingId = 'UA-68630113-1';
 
-  function injectAnalytics($) {
-    var analyticsHtml =
-    '\n<script>\n' +
-    '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' +
-    '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' +
-    'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' +
-    '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' +
-    'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' +
-    'ga(\'send\', \'pageview\');\n' +
-    '</script>\n';
-
-    $(head).append(analyticsHtml);
+  function injectAnalytics() {
+    var analytics = document.createElement("script");
+    analytics.textContent = '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' + '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' + 'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' + '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' + 'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' + 'ga(\'send\', \'pageview\');\n';
+    head.appendChild(analytics);
   }
 
-  function setupBar(err, $) {
-    if(err) {
+  function setupBar(err) {
+    if (err) {
       console.log("[Thimble Error] Unable to inject Remix UI. Error was: `" + err + "`");
       return;
     }
 
-    var isTouchDevice = 'ontouchstart' in document.documentElement;
-    var detailsBar = $(".details-bar");
-    detailsBar.attr("style", "");
-
-    if(isTouchDevice) {
-      detailsBar.addClass("touch-mode");
-    } else {
-      detailsBar.addClass("mouse-mode");
+    var detailsBar = document.querySelector(".details-bar");
+    
+    if (detailsBar) {
+      detailsBar.setAttribute("style", "");
+      detailsBar.classList.add("mouse-mode");
+
+      detailsBar.addEventListener("click", function(event) {
+        if (event.target.classList.contains("thimble-button")) {
+          detailsBar.classList.remove("collapsed");
+          event.stopPropagation();
```

Comment:
I was returning false previously but in an earlier review [Pomax suggested](https://github.com/mozilla/thimble.mozilla.org/pull/1825#discussion_r106050023) that HTML5 ignores `return false`. So I changed it to `event.stopPropagation()`